xml/article_nfs_storage.xml

```diff
@@ -166,47 +162,40 @@
    </listitem>
   </itemizedlist>
   <para>
-    As the LVM volume group is named <literal>nfs</literal>, the
-    DRBD resource uses the same name.
+    The following procedures will result in two DRBD devices: one device
+    for the NFS file system, and a second device to share the NFS file states.
    </para>
 
    <sect2 xml:id="sec-ha-quick-nfs-drbd-config">
-    <title>Creating DRBD configuration</title>
+    <title>Creating DRBD configurations</title>
     <para>
      For consistency reasons, it is highly recommended to follow this advice:
     </para>
-
     <itemizedlist>
      <listitem>
       <para>Use the directory <filename>/etc/drbd.d/</filename> for your
-      configuration.</para>
+      configurations.</para>
      </listitem>
      <listitem>
-      <para>Name the file according to the purpose of the resource.
+      <para>Name the files according to the purpose of the resources.
       </para>
      </listitem>
      <listitem>
-      <para>Put your resource configuration in a file with a <filename
-       class="extension">.res</filename> extension. In the following
-        examples, the file <filename>/etc/drbd.d/nfs.res</filename> is
-        used.
+      <para>Put your resource configurations in files with a <filename
+       class="extension">.res</filename> extension.
       </para>
      </listitem>
     </itemizedlist>
-
-    <para>
-     Proceed as follows:
-    </para>
     <procedure>
-     <title>Creating a DRBD configuration</title>
+     <title>Creating DRBD configurations</title>
      <step>
       <para>
-       Create the file <filename>/etc/drbd.d/nfs.res</filename> with the
+       Create the file <filename>/etc/drbd.d/nfs-exportfs.res</filename> with the
         following contents:
       </para>
-<screen>resource nfs {
-   device /dev/drbd0; <co xml:id="co-ha-quick-nfs-drbd-device"/>
-   disk   /dev/nfs/work; <co xml:id="co-ha-quick-nfs-drbd-disk"/>
+<screen>resource nfs-exportfs {
+   device /dev/drbd1 minor 1; <co xml:id="co-ha-quick-nfs-drbd-device"/>
+   disk   /dev/nfs/exportfs; <co xml:id="co-ha-quick-nfs-drbd-disk"/>
```

Comment:
@zzhou1 I changed the initial parts of the DRBD config, but left the rest the same as it already was (see https://documentation.suse.com/sle-ha/15-SP4/html/SLE-HA-all/article-nfs-storage.html#sec-ha-quick-nfs-drbd-config).

Let me know if I do need to include the extra details from the configs you attached in the bug.

Reply:
Sounds good to me. Instead of copy&paste among DOCs.